core/service-test-runner/service-tester.js

```diff
@@ -116,7 +116,7 @@ class ServiceTester {
    * @param {string} attrs.baseUrl base URL for test server
    * @param {boolean} attrs.skipIntercepted skip tests which intercept requests
```

Comment:
I don't recall offhand how to best handle a nested object param, but we should probably update the jsdoc here to account for the new retry param

Reply:
Thanks! https://jsdoc.app/tags-param.html describes this case:

> If a parameter is expected to have a specific property, you can document that property by providing an additional @param tag. For example, if an employee parameter is expected to have name and department properties, you can document it as follows:
> Documenting a parameter's properties
> 
> /**
>  * Assign the project to an employee.
>  * @param {Object} employee - The employee who is responsible for the project.
>  * @param {string} employee.name - The name of the employee.
>  * @param {string} employee.department - The employee's department.
>  */
> Project.prototype.assign = function(employee) {
>     // ...
> };

I expected ESLint+JSDoc plugin to catch it. But JSDoc plugin doesn't support a destructuring parameter: https://github.com/gajus/eslint-plugin-jsdoc/issues/11